Sampler/Templates/Build/build.yaml.template

```diff
@@ -66,11 +84,25 @@ NestedModule:
 ####################################################
 #       Sampler Pipeline Configuration             #
 ####################################################
+<%
+    if ($PLASTER_PARAM_ModuleType -eq 'dsccommunity') {
+@"
+BuildWorkflow:
+  '.':
+    - build
+    - test
```

Comment:
It looks like when the `ModuleType -eq 'dsccommunity'` we're not doing `hqrmtest` anymore, am I right? is that Intentional?

Reply:
It is doing hqrmtest in a job in the azure-pipelines, but not default when running task 'test'. Need to run `-Task hqrmtest` to run them from console. So it is intentionally as is this way in all the modules we converted.